package-lock.json

```diff
@@ -357,9 +357,9 @@
       },
       "dependencies": {
         "fs-capacitor": {
-          "version": "2.0.4",
-          "resolved": "https://registry.npmjs.org/fs-capacitor/-/fs-capacitor-2.0.4.tgz",
-          "integrity": "sha512-8S4f4WsCryNw2mJJchi46YgB6CR5Ze+4L1h8ewl9tEpL4SJ3ZO+c/bS4BWhB8bK+O3TMqhuZarTitd0S0eh2pA=="
+          "version": "6.2.0",
```

Comment:
💡These changes to package-lock didn't seems to affect me. However, I would still revert them just in case, since these updates don't seems all that useful/vital